<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowAddLeader</class>
 <widget class="QWidget" name="WindowAddLeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,4">
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineOrganization"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="linePhone"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Organization</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineSurname"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEmail"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineCity"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
